### DISCUSSION TITLE
How to deal with the error message on webstorm

### DISCUSSION THREAD

**Discussion by saddit** [posva/unplugin-vue-router#94] [Questions and Help] [answered]
![image](https://user-images.githubusercontent.com/55338151/201918360-68afe0ab-9946-4875-93bf-dddc1365b3dd.png)
![image](https://user-images.githubusercontent.com/55338151/201918614-fb22cd18-3dfc-4738-8229-451d729b92bf.png)
```
Unable to resolve directory 'auto'

Unable to resolve file 'auto'
```

**Comment by ojvribeiro** [comment]
On VSCode it's also unable to resolve the `auto` folder. I'm using Laravel Mix (Webpack 5).

**Comment by duclet** [comment, marked as answer]
Running into this same issue. What I do notice is that this problem only seem to happen on `*.vue` files but works fine in `*.ts` files. So my workaround is to import and re-export the things I need in a `.ts` file and import from that in `.vue` files. Obviously, this doesn't work for macros like definePage but I don't use that.
